docs/devGuide/devGuide.md

```diff
@@ -175,10 +175,42 @@ On Windows:
   You should always check that the generated output is correct before committing any changes to the test sites.
 </box>
 
-##### New features
+#### Adding test site content
 
 When adding new features, you should also add new site content into an existing test site or create a new test site to demonstrate the new feature. This is to ensure that your feature can be tested by building that test site.
 
+To add a page to an existing test site, for this example, to `test_site`:
+
+1. Add a new test page, e.g., `newTestPage.md`, containing a demonstration of the new feature.
+
+2. Open the `site.json` corresponding to the test site, i.e. `test/functional/test_site/site.json`
+
+3. To include the new page, i.e. `newTestPage.md`, add it to the `pages` array.
+
+``` {heading="site.json" highlight-lines="16,17"}
+"pages": [
+    {
+      "src": "index.md",
+      "title": "Hello World",
+      "frontmatter": {
+        "frontMatterOverrideProperty": "Overridden by front matter override",
+        "globalAndFrontMatterOverrideProperty":  "Overridden by front matter override"
+      }
+    },
+    ...
+    {
+      "src": "testLayouts.md",
+      "title": "Hello World"
+    },
+    {,
+      "src": "newTestPage.md",
+      "title": [some title you see fit]
+    },
+    ....
+```
+
+4. Update the tests.
+
 <box type="info">
   When creating a new test site, the directory name of the new test site should be added to <code>test/functional/test_sites</code>.
```

Comment:
Could remove this as well, we don't have a `test_sites` directory

Reply:
It's a file without an extension. Please check again :sweat_smile:

Reply:
https://github.com/MarkBind/markbind/tree/master/test/functional/test_sites

Perhaps we should mention that it's a file. Since you (and Tejas and even i) thought it was a directory at first.

Reply:
Ah my bad. I even added to it at one point ( `test_site_special_tags` ) 😅

Reply:
> Perhaps we should mention that it's a file. Since you (and Tejas and even i) thought it was a directory at first.

How to mention its a file though? 

Something like - When creating a new test site, the directory name of the new test site should be added to <code>test/functional/test_sites</code> **file**.